builtin/providers/aws/resource_aws_elastic_beanstalk_environment.go

```diff
@@ -56,6 +56,10 @@ func resourceAwsElasticBeanstalkEnvironment() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"version_label": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
I believe it does, at least when I change the value in my Terraform document, it is aware of the change when running `terraform plan`

Related to this, I did see a case where the version field wasn't being managed by Terraform. Subsequent plans would show the value changing from the external value to "". To work around that `ignore_changes` was used for that field. Changing `Computed: true` also fixed the issue (and removed the need for `ignore_changes`), but I wasn't sure if that was the intended behavior of that option. 

Not sure I completely understand the purpose of `Computed` yet. Based on the comments for `Computed` and some of the documentation (https://www.terraform.io/intro/getting-started/build.html) I thought it was for values that wouldn't be known until the resource was created. In this case we are providing the value in the Terraform document, so why would it be computed?